scripts/package-build

```diff
@@ -148,6 +148,16 @@ for pkgname in "${pkgnames[@]}"; do
             # Run packaging script
             package
 
+            section "Stripping $pkgname"
+            docker container run --interactive --rm \
+                --mount type=bind,src="$(realpath "$pkgdir")",dst=/pkg \
+                "$(image-name base)" /bin/bash \
+                << "SCRIPT"
+find /pkg -print0 -type f | xargs --null arm-linux-gnueabihf-strip --strip-all &> /dev/null || true
+SCRIPT
+            find "$pkgdir" -print0 -type f | xargs --null strip --strip-all &> /dev/null || true
```

Comment:
why is strip happening twice, once in docker and once not?

Reply:
One is using `arm-linux-gnueabihf-strip` to strip files in that format, the other uses `strip` to strip any x86 files also going along. This is necessary as some files are not in a format that `arm-linux-gnueabihf-strip` understands, but still can be stripped.